File: Sorting algorithms/selection.py
def selection(arr):
    for i in range(len(arr)):
        min = i
        for j in range(i+1,len(arr)):
            if arr[min]>arr[j]:
                arr[min], arr[j]=arr[j], arr[min]
    return arr

arr=[5,3,7,9,3,6,67,23,12,6,421,1,0]
print(selection(arr))

File: Sorting algorithms/selection.js
function selection(arr){
  for(i=0;i<=arr.length-1;i++){
    for(j=i+1;j<=arr.length-1;j++){
      if(arr[i]<arr[j]){
        continue;
      }
      else{
        [[arr[i]],[arr[j]]]=[[arr[j]],[arr[i]]];
      }
    }
  }
  return arr;
}

var arr = [456,23,888,123,34,33,1,0,45,99,320,50];
console.log("Here is unsort array: "+arr);
console.log("Here is sorted array: "+selection(arr));
